btree/btree.c

```diff
@@ -1885,12 +1869,45 @@ static int64_t mem_tree_get(struct node_op *op, struct segaddr *addr, int nxt);
  */
 static int64_t tree_get(struct node_op *op, struct segaddr *addr, int nxt)
 {
-	int        nxt_state;
+	struct td              *tree = NULL;
+	struct nd              *node = NULL;
+
+	if (addr != NULL) {
+		nxt  = node_get(op, NULL, addr, nxt);
```

Comment:
Need to handle error codes returned by node_get() .

Reply:
Yes. I missed that part.
I will update it.